requirements.txt

```
@@ -1,5 +1,6 @@
 Babel==2.9.1  # min version = 2.6.0 (Focal with security backports)
 chardet==3.0.4
+cryptography<37.0.0  # remove this when pyopenssl>=21.0.0
```

Please use equality only to prevent non determinism as much as possible.
So in this case, you should use the Debian [version](https://packages.debian.org/buster/python3-cryptography) from Buster
```suggestion
cryptography==2.6.1  # incompatibility between pyopenssl 19.0.0 and cryptography>=37.0.0
```

---

Done :)